catalyst_voices/lib/widgets/snackbar/voices_snackbar.dart

```diff
@@ -0,0 +1,146 @@
+import 'package:catalyst_voices/widgets/snackbar/voices_snackbar_type.dart';
+import 'package:catalyst_voices_assets/catalyst_voices_assets.dart';
+import 'package:catalyst_voices_brands/catalyst_voices_brands.dart';
+import 'package:catalyst_voices_localization/catalyst_voices_localization.dart';
+import 'package:flutter/material.dart';
+
+class VoicesSnackBar extends StatelessWidget {
+  final VoicesSnackBarType snackBarType;
+  final VoidCallback? onRefreshPressed;
+  final VoidCallback? onLearnMorePressed;
+  final VoidCallback? onOkPressed;
+  final VoidCallback? onClosePressed;
```

Comment:
I'm not sure if this approach with hardcoded callbacks is going to work, designers often change their minds,they rename labels, etc.

Usually I created a snackbar with primary and secondary action, then if you wanted to use it you would:

```
VoicesSnackBar(
  primaryAction: VoicesSnackBarAction(
    label: 'Refresh',
    onPressed: () {},
 ),
 secondaryAction: ...,
)
```

This way you can handle different types of actions, not only the ones that were hardcoded initially.

Reply:
I also not sure about that :)